eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -252,7 +258,13 @@ jobs:
         extraBuildProperties="$extraBuildProperties /p:OverrideTargetRid=${{ parameters.targetRid }}"
       fi
 
-      docker run --rm -v "$(sourcesPath):/vmr" -w /vmr $customRunArgs ${{ parameters.container }} ./build.sh --clean-while-building $(additionalBuildArgs) $customBuildArgs $extraBuildProperties
+      # Only use Docker stuff on Linux
+      if [[ "`uname -s`" == "Linux" ]]; then
+        docker run --rm -v "$(sourcesPath):/vmr" -w /vmr $customRunArgs ${{ parameters.container }} ./build.sh --clean-while-building $(additionalBuildArgs) $customBuildArgs $extraBuildProperties
+      else
+        cd $(sourcesPath)
+        CROSSCOMPILE=1 ./build.sh --clean-while-building $(additionalBuildArgs) $customBuildArgs $extraBuildProperties
```

Comment:
I'd prefer setting this similar to how we do it in the if that checks parameters.crossRootFs above, via customRunArgs.

Maybe refactor that if so it applies to cross build with and without RootFs.